src/main/resources/db/changelog/db.changelog-master.yaml

```
@@ -75,4 +75,4 @@ databaseChangeLog:
     file: v1.22.0/v1.22.0.yaml
 - include:
     relativeToChangelogFile: true
-    file: v1.23.0/v1.23.0.yaml
+    file: v1.23.0/v1.23.0.yaml
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)